src/pages/home/Home.jsx

```diff
@@ -1,12 +1,59 @@
 import axios from "axios";
 import React from "react";
 import { useEffect, useState } from "react";
+import { useNavigate } from "react-router-dom";
+import { useCategories } from "../../context/categories-context";
+import {ImageSlider} from "../../components/ImageSlider/ImageSlider";
 import "./home.css";
+import { useAuth } from "../../context/auth-context";
 
 export const Home = () => {
+  const { categoryState } = useCategories();
+  const navigate = useNavigate();
+  const  {isLoggedIn} = useAuth()
+  const allSubCategories = categoryState.allCategoryDetails.reduce(
+    (acc, category) => [...acc, ...category.subCategories],
+    []
+  );
+  const featuredCategories = allSubCategories.filter(
+    (subCategory) => subCategory.featured
+  );
+
+  const { toggleSubCategoryState, resetCategories} = useCategories();
+  const featuredCategoryHandler = (id) => {
+    toggleSubCategoryState(id);
+    navigate("/product-listing");
+  };
+  useEffect(() => resetCategories(), [])
   return (
-    <div className="home">
-      <p>This is Home</p>
-    </div>
+    <>
+      <h2>Meltin' Moments</h2>
+      <section>
+        <ImageSlider />
+        <h3>Featured Categories</h3>
+        <div className="grid-container grid-fit">
+          {featuredCategories.map(
+            ({ categoryName, _id, media, description }) => (
+              <div
+                className="card-container card-vertical grid-child"
+                title={categoryName}
+                key={_id}
```

Comment:
Appreciation: you have destructured the map element before passing it to JSX. Great work.